js/rollup.config.js

```diff
@@ -7,7 +7,7 @@ import tailwindcss from 'tailwindcss'
 import typescript from '@rollup/plugin-typescript'
 import { typescriptPaths } from 'rollup-plugin-typescript-paths'
 import replace from '@rollup/plugin-replace';
-import pkg from './package.json' assert { type: 'json' };
+import pkg from './package.json' with { type: 'json' };
```

Comment:
this change is related to node version, we will go with the 'assert' instead of 'with', so this need not be merged.